Makefile

```diff
@@ -28,7 +28,7 @@ PACKAGE_URL := https://www.getlantern.org
 
 LANTERN_BINARIES_PATH ?= ../lantern-binaries
 
-GO_MOBILE_REVISION=754ad3cfda342ad8c5d07a5ef9334f8a62b6aefa
+GO_MOBILE_REVISION=47553f4d4275316f2610b3204f8f9ecaf3ce63a7
```

Comment:
A revision from April, around the time I first commited the change that introduced the `libgojni.so` compilation from the `Makefile`.